.travis.yml

```diff
@@ -5,7 +5,6 @@ branches:
 script:
   - bundle exec rake
 rvm:
-  - 2.0.0-p648
-  - 2.1.9
-  - 2.2.5
-  - 2.3.1
+  - 2.2
+  - 2.3
+  - 2.4
```

Comment:
What version is chosen when `2.2` is selected vs `2.2.5`?

Reply:
As I understand, pre-installed Ruby versions in Travis CI. [Some documentation](https://docs.travis-ci.com/user/languages/ruby/#Specifying-Ruby-versions-and-implementations). And in my experience it's close to the last released patch-version (with a slight lag). I don't see the need to specify the patch-version until there is something specific (bugs in the pre-installed version of Ruby in Travis CI, for example: bug in 2.4.1, Travis CI has 2.4.1 pre-installed for `2.4`, but `2.4.2` is already available).